ansible/group_vars/all/inv_mapping.yml

```diff
@@ -0,0 +1,8 @@
+---
+# Specify all public mappings below
+lab: lab_connection_graph.xml
+inventory: lab_connection_graph.xml
```

Comment:
Sorry missed this one. Why do we need to have inventory file here?

@neethajohn , @wangxin comments?

Reply:
@neethajohn @wangxin could you please reply to this comment?

Reply:
@neethajohn @wangxin could you please reply to this comment?

Reply:
@yvolynets-mlnx , I guess Ying's question is why there is a key named 'inventory' in that file mapping to 'lab_connection_graph.xml'.

Reply:
Different inventory file names can have the same graph file mapping.
So "inventory" file also was added here, as it is one of main inventory files.

Reply:
Yuri, I don't think we have an inventory called 'inventory' unless it is something Mellanox specific? If so, can you add it in your internal repo instead?

I am ok with merging this PR first and continue discussion and make fix/updates later because I believe this change won't break anything.

Reply:
Yes we can add this "inventory" key internally. However there is the same file in master as well, which is updated to our needs internally - https://github.com/Azure/sonic-mgmt/blob/master/ansible/inventory.
I suppose some other user also could use this file as it does exist for a long time.